Templates/Full/game/core/scripts/client/defaults.cs

```diff
@@ -436,11 +457,34 @@ class = "GraphicsQualityLevel";
       key["$pref::Video::disablePixSpecular"] = false;
       key["$pref::Video::disableNormalmapping"] = false;
       key["$pref::Video::disableParallaxMapping"] = false;     
-      key["$pref::Water::disableTrueReflections"] = false;          
-   };   
+      key["$pref::Water::disableTrueReflections"] = false; 
+      key["$pref::Video::disableSSAO"] = true;
+      key["$pref::Video::disableHDR"] = false;
+      key["$pref::Video::disableVignette"] = false;
+      key["$pref::Video::disableDOF"] = false;
+      key["$pref::Video::disableLightRays"] = false;
+   };
+   new ArrayObject( [Highest] )
+   {
+      class = "GraphicsQualityLevel";
+      caseSensitive = true;
+      
+      key["$pref::Video::disablePixSpecular"] = false;
+      key["$pref::Video::disableNormalmapping"] = false;
+      key["$pref::Video::disableParallaxMapping"] = false;     
+      key["$pref::Water::disableTrueReflections"] = false; 
+      key["$pref::Video::disableSSAO"] = false;
+      key["$pref::Video::disableHDR"] = false;
+      key["$pref::Video::disableVignette"] = false;
+      key["$pref::Video::disableDOF"] = false;
+      key["$pref::Video::disableLightRays"] = false;
+   };
 };
 
-
+function ShaderQualityGroup::onApply( %this, %level )
+{
+   PostFXManager.settingsSetEnabled(true);
```

Comment:
This bit came up for discussion RE: https://github.com/GarageGames/Torque3D/issues/679
intended to work in conjunction with https://github.com/GarageGames/Torque3D/blob/39f0e269d6c4f653f6424c70bc580e64545a9d7d/Templates/Empty/game/core/scripts/client/postFx/postFxManager.gui.settings.cs#L39 among other things to resize the backbuffer *before* you load into game and have to do it anyway per-level.